doc/contribute/proposals_and_rfcs.rst

```diff
@@ -40,16 +40,17 @@ Some changes do not require an RFC:
 - Addition of new boards, SoCs or drivers to existing subsystems
 - ...
 
-The process in itself consists in creating a GitHub issue with the :ref:`RFC
-label <gh_labels>` that documents the proposal thoroughly. There is an `RFC
-template`_ included in the main Zephyr GitHub repository that serves as a
-guideline to write a new RFC.
+The process in itself consists in `creating a GitHub issue`_ with the `RFC label`_
```

Comment:
not really blocking but I think I would rather stick to the original link for the "RFC label" or, maybe better, word it as: "... with the :ref:`RFC
label <gh_labels>` (see currently open RFCs `here`_)" with 'here" linking to https://github.com/zephyrproject-rtos/zephyr/issues?q=is%3Aissue+label%3ARFC+is%3Aopen ?